db/migrate/20200425194536_remove_unused_profile_flags.rb

```diff
@@ -0,0 +1,10 @@
+class RemoveUnusedProfileFlags < ActiveRecord::Migration[5.2]
+  def change
```

Comment:
This change will break:
- data export
- the "Your data" view in the settings page
- everything else that makes use of them